### PR TITLE
MazeGen styles

### PR DESCRIPTION
**Pull request by LisanneW**
Fix exitScore incorrectly updated, could be overridden by lower scores.
Difficulty levels now have different styles and level sizes.

### CODE REVIEW

File: MeesGame/Gameplay/Generator/MazeGeneratorInit.cs [hunk 1]
```diff
@@ -145,7 +97,7 @@ private void InitialiseKeys()
             {
                 do
                 {
-                    keyNodes[i] = (random.Next(totalNodesToDo - 1) + random.Next(totalNodesToDo - 1)) / 2 + 1; // Try to get it halfway.
+                    keyNodes[i] = random.Next(totalNodesToDo - 1) + 1;
```
Comment (maxmeijer42) [inline MeesGame/Gameplay/Generator/MazeGeneratorInit.cs]: Why the change?

Comment (jensj12) [reply MeesGame/Gameplay/Generator/MazeGeneratorInit.cs]: To spread the keys more evenly over the maze. Still not sure what the best formula would be, but this one isn't going to stay either. Something like 20/80 instead of 50/50 (current) or 100/0 (proposed change) may work better. I could even make it a difficulty setting.
